# Command the robot to capture simultaneous RGB and depth images from the specified camera.
string camera_name
---
sensor_msgs/CameraInfo info
sensor_msgs/Image rgb
sensor_msgs/Image depth
